src/components/flowchart/draw.js

```diff
@@ -301,12 +320,17 @@ export const drawEdges = function (changed) {
   const exitEdges = this.el.edges.exit();
   const allEdges = this.el.edges.merge(enterEdges).merge(exitEdges);
 
-  if (changed('edges')) {
-    enterEdges
-      .append('path')
+  if (changed('edges', 'newParamsFlag', 'focusMode', 'inputOutputDataNodes')) {
+    enterEdges.append('path');
+    allEdges
+      .select('path')
       .attr('marker-end', (edge) =>
         edge.sourceNode.type === 'parameters'
-          ? `url(#pipeline-arrowhead--accent)`
+          ? focusMode !== null && inputOutputDataEdges[edge.id]
```

Comment:
nit: I think giving `focusMode !== null && inputOutputDataEdges[edge.id]` a name might help with this block as it's repeated twice in this nested tenary operator. You can also reuse that variable further down as well.

Reply:
sounds good, I'll extract that out.